HISTORY.md

```diff
@@ -1,5 +1,20 @@
 # Release History - Autonolas Documentation
 
+## v0.10.0.post1 (2023-04-03)
+
+Frameworks
+
+* Open Autonomy Framework: open-autonomy https://github.com/valory-xyz/open-autonomy (v0.10.0.post2)
+
+Toolkits
+
+* SMPKit: autonomous-fund - https://github.com/valory-xyz/autonomous-fund (v0.5.0.post1)
+* MintKit: agent-academy-1 - https://github.com/valory-xyz/agent-academy-1 (v0.8.0.post1-3-g116a585)
```

Comment:
why is the tag so long?

Reply:
It refers to a not-mergeable branch with a ligtweight fix (documentation) after the release took place.

Reply:
This is a different issue.

It is fixed in this PR https://github.com/valory-xyz/agent-academy-1/pull/200/ but hasn't been merged before the release. Thus, there is a DNM branch rooted on the last release that implements this fix in the documentation.